test/http_client_close_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:cronet/cronet.dart';
+import 'package:test/test.dart';
+
+int main() {
+  late HttpServer server;
+  final sentData = 'Hello, world!';
+  setUp(() async {
```

Comment:
You should consider wrapping all your tests inside groups. They are printed as prefix for every test inside. You can simply use the name of your test file here without the extension.

Example test from ffigen - https://github.com/dart-lang/ffigen/blob/master/test/header_parser_tests/functions_test.dart#L17.